src/Bundle/test/src/Tests/DependencyInjection/SyliusMailerExtensionTest.php

```diff
@@ -91,6 +92,55 @@ public function it_configures_twig_and_swiftmailer_adapters_if_they_are_availabl
         $this->assertContainerBuilderHasAlias('sylius.email_renderer.adapter', 'sylius.email_renderer.adapter.twig');
     }
 
+    /** @test */
+    public function it_configures_twig_and_symfony_mailer_adapters_if_they_are_available(): void
+    {
+        $this->mockService('event_dispatcher', EventDispatcher::class);
+        $this->mockService('mailer.mailer', MailerInterface::class);
+        $this->mockService('twig', Environment::class);
+
+        $this->container->setParameter(
+            'kernel.bundles',
+            ['TwigBundle' => TwigBundle::class]
+        );
+
+        $this->load();
+        $this->compile();
+
+        $this->assertContainerBuilderHasParameter('sylius.mailer.sender_name', 'Example.com Store');
+        $this->assertContainerBuilderHasParameter('sylius.mailer.sender_address', 'no-reply@example.com');
+
+        $this->assertContainerBuilderHasService('sylius.email_sender.adapter.symfony_mailer', SymfonyMailerAdapter::class);
+        $this->assertContainerBuilderHasAlias('sylius.email_sender.adapter', 'sylius.email_sender.adapter.symfony_mailer');
+
+        $this->assertContainerBuilderHasService('sylius.email_renderer.adapter.twig', EmailTwigAdapter::class);
+        $this->assertContainerBuilderHasAlias('sylius.email_renderer.adapter', 'sylius.email_renderer.adapter.twig');
+    }
+
+    /** @test */
+    public function it_configures_twig_and_swift_mailer_adapters_over_symfony_mailer_if_both_are_available(): void
+    {
+        $this->mockService('event_dispatcher', EventDispatcher::class);
+        $this->mockService('mailer.mailer', MailerInterface::class);
+        $this->mockService('swiftmailer.mailer.default', \Swift_Mailer::class);
+        $this->mockService('twig', Environment::class);
+
+        $this->container->setParameter(
+            'kernel.bundles',
+            ['SwiftmailerBundle' => SwiftmailerBundle::class, 'TwigBundle' => TwigBundle::class]
+        );
+
+        $this->load();
+        $this->compile();
+
+        $this->assertTrue(interface_exists(MailerInterface::class));
+        $this->assertContainerBuilderHasService('sylius.email_sender.adapter.symfony_mailer', SymfonyMailerAdapter::class);
+        $this->assertContainerBuilderHasAlias('sylius.email_sender.adapter', 'sylius.email_sender.adapter.symfony_mailer');
```

Comment:
Seems wrong if I'm correct

Reply:
I have updated the function name.

I made this test because I want to be sure that SymfonyMailer will be used if SwiftMailer is also present in the project (because SwiftMailer is on Sylius/Sylius Standard and SymfonyMailer is installed by the developer).

Reply:
IMHO, we should cover this other way around. Right now, one could add a Symfony mailer to their end Sylius application(to use it for other mailings). If we will merge it with the current state, the application may not work properly(due to some changes, as the default adapter has been switched. It may be considered BC Break. 

However, usage of SymfonyMailer by default may not be a bad option, if we will mention this change in UPGRADE file.